content/general/concepts/modulo/modulo.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Modulo'
+Description: 'A modulo operation takes two arguments and returns the remainder of the first argument divided by the second.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arithmetic'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Modulo is a mathematical operation available in every programming language along with more common ones like addition and multiplication. A modulo operation takes two arguments and returns the remainder of the first argument divided by the second. The result of the operation is called the modulus.
+
+Most programming languages use the symbol `%` to represent the modulo operation, though some may use `mod` or other variations.
+
+## Examples
+
+The following is an example of the modulo operator in Python:
+
+```py
+modulus = 14 % 5
+print(f"14 % 5 results in a modulus of {modulus}")
+# Output : 14 % 5 results in a modulus of 4
+
+modulus = 37 % 3
+print(f"37 % 3 Results in a modulus of {modulus}")
+# Output: 37 % 3 Results in a modulus of 1
+
+modulus = 21 % 7
+print(f"21 % 7 results in a modulus of {modulus}")
+# Output: 21 % 7 results in a modulus of 0
+```
```

Comment:
```suggestion
```codebyte/py
modulus = 14 % 5
print(f"14 % 5 results in a modulus of {modulus}")

modulus = 37 % 3
print(f"37 % 3 Results in a modulus of {modulus}")

modulus = 21 % 7
print(f"21 % 7 results in a modulus of {modulus}")
```
```